pkg/kubefed2/federate/disable.go

```diff
@@ -121,12 +121,7 @@ func (j *disableType) Run(cmdOut io.Writer, config util.FedConfig) error {
 		Name:      j.targetName,
 	}
 
-	err = DisableFederation(cmdOut, hostConfig, typeConfigName, j.delete, j.DryRun)
-	if err != nil {
-		return err
-	}
-
-	return nil
+	return DisableFederation(cmdOut, hostConfig, typeConfigName, j.delete, j.DryRun)
```

Comment:
(No action required) I think this kind of change is of questionable value.  It might look cleaner, but if there are more than one return statement in a function - as is the case in most of these changes in this PR - I think having a default return statement may well be preferable:
 
 - new code can be added after the last error-checked statement without modifying the last statement
 - the last error-checked statement can be removed without further modification (i.e. adding a return statement) 

I'm not suggesting code should be one way or the other, though, and I wouldn't reject code submitted in either style.  Similarly, I don't think there is a basis for 'cleanup' of either form.  Whoever submits a given piece of functionality gets to decide what they think is best, and maintainers making functional changes can modify that initial decision if it makes sense to do so.